src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -485,47 +482,66 @@ export default class GroupComparisonStore {
     }
 
```

Comment:
please go through and delete any members that are now unused. eg I know that enrichmentsGroup1 and 2 are now unused

Reply:
seems still used for mrna and protein enrichments

Reply:
oh true